### ISSUE TITLE
RVVcode - skos?

### ISSUE THREAD

**Issue by ElisabethDeVries** [Stichting-CROW/verkeersborden#365]

**Comment by gulaycanbaloglu** [comment]
Exactly, we will use the skos:notation for the rvv-code in the stylesheet.

**Comment by ElisabethDeVries** [comment]
do we change it in LACES? To be able tho use a standard stylesheet

**Comment by ElisabethDeVries** [comment]
En wordt de toelichting op de verkeersmaatregel ook een skos:note?

**Comment by ElisabethDeVries** [comment]
> En wordt de toelichting op de verkeersmaatregel ook een skos:note?

Ja, maar dit bestaat niet in het informatiemodel, maar kan worden gebruikt bij individuele maatregelen, dus kan worden gebruikt door de wegbeheerder. Toevoegen als toelichting aan de databeheerder
